modules/svg/SCsub

```diff
@@ -44,22 +43,29 @@ thor_src = [
     "src/lib/sw_engine/tvgSwShape.cpp",
     "src/lib/sw_engine/tvgSwStroke.cpp",
 ]
-thirdparty_sources = [thirdparty_dir + file for file in thor_src]
-env_svg.Prepend(CPPPATH=["."])
-env_svg.Prepend(CPPPATH=["#thirdparty/thorvg/src/lib/"])
-env_svg.Prepend(CPPPATH=["#thirdparty/thorvg/src/lib/sw_engine/"])
-env_svg.Prepend(CPPPATH=["#thirdparty/thorvg/src/loaders/raw/"])
-env_svg.Prepend(CPPPATH=["#thirdparty/thorvg/src/loaders/svg/"])
-env_svg.Prepend(CPPPATH=["#thirdparty/thorvg/src/loaders/png/"])
-env_svg.Prepend(CPPPATH=["#thirdparty/thorvg/inc/"])
+thirdparty_sources = [thirdparty_dir + file for file in thirdparty_sources]
+
+env_svg.Prepend(CPPPATH=[thirdparty_dir + "/inc"])
+
+env_thirdparty = env_svg.Clone()
+env_thirdparty.disable_warnings()
+env_thirdparty.Prepend(
+    CPPPATH=[
+        thirdparty_dir + "/src/lib",
+        thirdparty_dir + "/src/lib/sw_engine",
+        thirdparty_dir + "/src/loaders/raw",
+        thirdparty_dir + "/src/loaders/svg",
+        thirdparty_dir + "/src/loaders/png",
+    ]
+)
+env_thirdparty.add_source_files(thirdparty_obj, thirdparty_sources)
+env.modules_sources += thirdparty_obj
 
 # Godot source files
 
 module_obj = []
 
 env_svg.add_source_files(module_obj, "*.cpp")
-env_svg.disable_warnings()
```

Comment:
You should not disable warnings on Godot code - that's why we always build thirdparty code in a dedicated cloned environment for itself.

Reply:
This was an oversight.